jdk-wrapper.sh

```diff
@@ -189,6 +189,10 @@ extension="tar.gz"
 if [ "${JDKW_PLATFORM}" = "macosx-x64" ]; then
   extension="dmg"
 fi
+# JDK 6 and before use .bin extension, note no specific macosx download available for these versions (using linux)
```

Comment:
So you're saying versions equal to and prior to JDK6 for Linux only used a bin format? And only a bin format?

It looks like it was a bin but that also wrapped an rpm:

http://www.oracle.com/technetwork/java/javase/downloads/java-archive-downloads-javase6-419409.html

Can you make the comment a little clearer?